pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -155,9 +155,9 @@ cassowary = buildLuarocksPackage {
   propagatedBuildInputs = [ lua penlight ];
 
   meta = with lib; {
-    homepage = "https://github.com/sile-typesetter/cassowary.lua";
-    description = "The cassowary constraint solver";
-    maintainers = with maintainers; [ marsam alerque ];
+    homepage = "https://github.com/simoncozens/cassowary.lua";
```

Comment:
this redirects to https://github.com/sile-typesetter/cassowary.lua 
This was changed in e1f6f37dc9c7477bf514447fc4807e1dd4be68ce and wrongly merged because one should edit those in maintainers/scripts/luarocks-packages.csv instead. Would you mind porting the changes to the csv file instead ?

Reply:
cc @alerque

Reply:
Sure can do. Sorry I missed this ping earlier.

Reply:
I've added you as a maintainer already, the homepage is generated via the rockspec: we can override it but would be best to fix the upstream rockspec.

Reply:
Got it. I was just going to ask where the URL was! The rockspec has been fixed, there just hasn't been a release since because there haven't been code changes. We'll do that eventually, thanks for the pointers.